src/vm/util.cpp

```diff
@@ -1817,11 +1781,46 @@ size_t GetCacheSizePerLogicalCpu(BOOL bTrueSize)
     {
     }
     PAL_ENDTRY
-#else
+
+    return param.maxSize;
+}
+#endif // _TARGET_X86_
+
+// fix this if/when AMD does multicore or SMT
```

Comment:
This comment is very obsolete. I was just about to ask about AMD support. I've found the following CPUID specification from AMD that contains all the necessary info here: https://www.amd.com/system/files/TechDocs/25481.pdf